Apps/Admin/Client/Utils/AgentUtility.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace HealthGateway.Admin.Client.Utils
+{
+    using HealthGateway.Admin.Common.Constants;
+
+    /// <summary>
+    /// Utilities for interacting with admin agents.
+    /// </summary>
+    public static class AgentUtility
+    {
+        /// <summary>
+        /// Returns the formatted representation of an agent's identity provider.
+        /// </summary>
+        /// <param name="identityProvider">The agent's identity provider.</param>
+        /// <returns>A string containing the formatted representation of a agent's identity provider.</returns>
+        public static string FormatKeycloakIdentityProvider(KeycloakIdentityProvider identityProvider)
+        {
+            return identityProvider switch
+            {
+                KeycloakIdentityProvider.Idir => "IDIR",
+                KeycloakIdentityProvider.PhsaAzure => "PHSA",
+                _ => identityProvider.ToString(),
```

Comment:
Is this safe? Should this blow up?

Reply:
Leaving as-is since this is just used to display a more nicely formatted version of the value, and it's only being seen on Admin.